packages/cicero-core/src/templatesaver.js

```diff
@@ -49,6 +50,8 @@ class TemplateSaver {
             dir: true
         }));
 
+        zip.file('logo.png', logoBuffer, options)
```

Comment:
Here you will need to read the logo from the `Metadata` class for the template and write it out into the zip archive.